packages/orm/src/serde.ts

```diff
@@ -49,8 +50,15 @@ export class PrimitiveSerde implements FieldSerde {
     data[this.fieldName] = maybeNullToUndefined(row[this.columnName]);
   }
 
-  dbValue(data: any) {
-    return data[this.fieldName];
+  dbValue(data: any, knex?: Knex) {
```

Comment:
Totally get why you're doing this, but coupling the serde stuff to a specific driver/connection like knex is unfortunate... It'd be doable for now, but something that we might eventually have to bake into our driver api, like `driver.to_tsvector(...)` so that the serdes could defer to driver-specific/non-knex implementations.

Wouldn't block this initial PR though.

Reply:
yeah maybe we skip doing this work for now - it's not as clean of a solution as I was hoping for. We could give that js lib a try outside joist 🤔

Reply:
Actually looking closer at the lib it doesn't look like a real implementation: https://github.com/lblblong/to-tsvector/blob/master/src/index.ts

It's adding the word positions, but not doing any dictionary stemming

Reply:
Shoot! :thinking:

Reply:
If we can't end up doing JS-side stemming, I think we'll need to update this code to try and recognize an already-stemmed value, and then _not_ wrap it in `to_tsvector`...

It looks like a pretty recognizable format, but still is a little janky.